### ISSUE TITLE
How do I resume training from a saved checkpoint?

### ISSUE THREAD

**Issue by daegonYu** [FlagOpen/FlagEmbedding#1128]
I have fine-tuned the reranker through this github. I would like to continue learning from the saved checkpoint. Can you give me some instructions on how to do that?

**Comment by staoxiao** [comment]
You can pass the fine-tuned model to `--model_name_or_path `

**Comment by daegonYu** [comment]
This method is not a method to continue learning from the model's interrupted LR and train dataset, but a method to additionally learn from an initialized state, right?

**Comment by staoxiao** [comment]
> This method is not a method to continue learning from the model's interrupted LR and train dataset, but a method to additionally learn from an initialized state, right?

Yes. The current script doesn't implement the function to resume training from the previous state.

**Comment by daegonYu** [comment]
Thank you for your reply.
